File: example/two_consumer_one_producer/common-person-client/src/main/java/br/com/zup/pact/commonperson/config/ClientBeans.java
package br.com.zup.pact.commonperson.config;

import com.google.gson.Gson;
import org.springframework.context.annotation.Bean;
import org.springframework.context.annotation.Configuration;
import org.springframework.web.client.RestTemplate;


@Configuration
public class ClientBeans {

    @Bean
    public Gson gson() {
        return new Gson();
    }

    @Bean
    public RestTemplate getRestTemplate() {
        return new RestTemplate();
    }
}

File: example/kotlin/one-consumer-one-provider/consumer/consumer/src/main/kotlin/br/com/zup/pact/consumer/config/ClientBeans.kt
package br.com.zup.pact.consumer.config

import org.springframework.boot.json.GsonJsonParser
import org.springframework.context.annotation.Bean
import org.springframework.context.annotation.Configuration
import org.springframework.web.client.RestTemplate

@Configuration
class ClientBeans {
//    @Bean
//    fun gson(): GsonJsonParser {
//        return GsonJsonParser()
//    }

    @Bean
    fun getRestTemplate(): RestTemplate {
        return RestTemplate()
    }
}